.env.example

```diff
@@ -0,0 +1 @@
+URL="http://localhost:7777"
```

Comment:
please make two variables URL and PORT and use PORT in docker-compose
[exemple](https://stackoverflow.com/questions/29377853/how-can-i-use-environment-variables-in-docker-compose)